bigbluebutton-tests/playwright/options/options.js

```diff
@@ -159,6 +160,72 @@ class Options extends MultiUsers {
       await expect(modPageLocator, 'should the meeting display the font size increased').toHaveScreenshot('moderator-page-font-size.png', screenshotOptions);
     }
   }
+
+  async enableOtherParticipantsWebcams() {
+    await this.modPage.waitForSelector(e.whiteboard);
+    await this.userPage.waitForSelector(e.whiteboard);
+    await this.modPage.shareWebcam();
+    await this.userPage.shareWebcam();
+    await sleep(500)
+    await this.modPage.hasNElements('video', 2, 'should display 2 video elements for the moderator');
+    await this.userPage.hasNElements('video', 2, 'should display 2 video elements for the attendee');
+
+    await openSettings(this.modPage);
+    await openSettings(this.userPage);
+    await this.modPage.waitAndClick(e.dataSavingsTab);
+    await this.userPage.waitAndClick(e.dataSavingsTab);
+    await this.modPage.hasElementChecked(e.enableWebcamsToggleBtn, 'checked', 'should display the toggle button as ON');
+    await this.userPage.waitAndClickElement(e.enableWebcamsToggleBtn);
+    await this.userPage.hasElementNotChecked(e.enableWebcamsToggleBtn, 'unchecked', 'should display the toggle button as OFF');
+    await this.modPage.waitAndClick(e.modalConfirmButton);
+    await this.userPage.waitAndClick(e.modalConfirmButton);
+    await sleep(500);
+
+    await this.userPage.hasNElements('video', 1, 'should display 1 video elements for the attendee');
```

Comment:
prefer to use `hasElementCount` for generic assertions;

`hasNElements` is intended for cases where there might be more elements than the expected in the DOM, but some are not visible - then we check the visible ones only

```suggestion
    await this.userPage.hasElementCount('video', 1, 'should display 1 video element for the attendee');
```